tests/integration/test_subscriptions_cli.py

```diff
@@ -11,32 +11,14 @@
 
 """
 
-import itertools
 import json
 
-import click
 from click.testing import CliRunner
-
 import pytest
 
-import planet
 from planet.cli import cli
-from planet.cli.cmds import coro, translate_exceptions
-from planet.cli.io import echo_json
-from planet.exceptions import PlanetError
-
-
-@pytest.fixture
-def all_subs():
-    """A collection of fake subscriptions for testing.
-
-    We're not testing the actual API workflow. All subscription states
-    are equal as far as the initial tests are concerned. We have 101
-    fake subscriptions so that we can test the command's default limit
-    of 100.
-
-    """
-    return [{'id': str(i), 'status': 'created'} for i in range(101)]
```

Comment:
This fixture is gone, it's been changed into planet.cli.subscription's `_fake_subs` attribute.